homeassistant/components/nibe_heatpump/button.py

```diff
@@ -0,0 +1,63 @@
+"""The Nibe Heat Pump sensors."""
+from __future__ import annotations
+
+from nibe.coil_groups import UNIT_COILGROUPS, UnitCoilGroup
+from nibe.exceptions import CoilNotFoundException
+
+from homeassistant.components.button import ButtonEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import DOMAIN, LOGGER, Coordinator
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up platform."""
+
+    coordinator: Coordinator = hass.data[DOMAIN][config_entry.entry_id]
+
+    def reset_buttons():
+        for entity_description in UNIT_COILGROUPS.get(coordinator.series, {}).get(
+            "main"
+        ):
+            try:
+                yield NibeAlarmResetButton(coordinator, entity_description)
```

Comment:
It's confusing to name the `UnitCoilGroup` parameter `entity_description` which we normally use for `EntityDescription` instances.

Reply:
Right. Remnant of old solution. Will fix.